### ISSUE TITLE
Abort trap: 6 with bad .pill

### ISSUE THREAD

**Issue by famousj** [urbit/urbit#1162]
When I try to boot up an arvo with a bad .pill file, the following happens.  The pill is bad and this shouldn't work, but it should exit with a bit more guidance than this:

```
$ urbit -B urbit-0.7.0-VERYBAD.pill -w ~naphex-mogmyr-ribben-donnyl -k ~naphex-mogmyr-ribben-donnyl.key
~
urbit 0.7.0
boot: home is /Users/leblancj/urbit/naphex-mogmyr-ribben-donnyl
loom: mapped 2048MB
boot: installed 256 jets
cp urbit-0.7.0-VERYBAD.pill naphex-mogmyr-ribben-donnyl/.urb/urbit.pill
boot: loading naphex-mogmyr-ribben-donnyl/.urb/urbit.pill
Abort trap: 6 (core dumped)
```

Longer story:
My laptop has a very zealous firewall which doesn't like whatever certificate `bootstrap.urbit.org` is using.  First time I tried to create a moon on the laptop, it told me to download the file myself.

I thought I'd just run `wget --no-check-certificate`.  This seemed to have worked, but the .pill file it created was an HTML error message.  I didn't realize this until I got the above-mentioned `Abort trap: 6 (core dumped)` and investigated.

Doesn't seem like a particularly high priority.  If y'all would like, I could look into this myself and submit a PR.

**Comment by ohAitch** [comment]
Iirc the pill format is [nock (list event)] or [(list event) nock] or sth;
prepending the head with [%10 %pill ...] or [[%pill ~] ...] respectively,
so the first bytes of jam could be used as a stable "magic number", would
probably be good.

(and in the case of seeing a <4k valid-utf8 file instead, printing that
out, since we've seen HTML errors sneaking in as pills on a number of
occasions now)

**Comment by ohAitch** [comment]
…%11*

/approximately/ stable magic number, then

**Comment by joemfb** [comment]
fixed on `hotfix` in #1172
